### PR TITLE
Add open graph metadata and missing assets

### PR DESCRIPTION
**Pull request by Windsdon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,12 @@
 		<title>Minecraft Double Smooth Stone Slab</title>
 		<link rel="shortcut icon" type="image/png" href="/assets/favicon.png" />
 		<link rel="icon" type="image/png" href="/assets/favicon.png" />
+		<meta property="og:title" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:site_name" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:url" content="https://minecraftdoublesmoothstoneslab.website">
+		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:type" content="article">
+		<meta property="og:image" content="https://minecraftdoublesmoothstoneslab.website/assets/minecraftdoublesmoothstoneslab.jpg">
```
Comment (hedgehog1029) [inline index.html]: this asset should be at the site root so we have a dumb link
```suggestion
		<meta property="og:image" content="/minecraftdoublesmoothstoneslab.jpg">
```

File: index.html [hunk 2]
```diff
@@ -4,6 +4,12 @@
 		<title>Minecraft Double Smooth Stone Slab</title>
 		<link rel="shortcut icon" type="image/png" href="/assets/favicon.png" />
 		<link rel="icon" type="image/png" href="/assets/favicon.png" />
+		<meta property="og:title" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:site_name" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:url" content="https://minecraftdoublesmoothstoneslab.website">
+		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:type" content="article">
```
Comment (hedgehog1029) [inline index.html]: this property isn't actually required but it should probably just be "website"?
```suggestion
		<meta property="og:type" content="website">
```

File: index.html (hunk 1 above)
Comment (hedgehog1029) [reply index.html]: actually discuss with me if this should be in assets or not, I'm not sure if I care about having the root link

File: index.html (hunk 2 above)
Comment (Windsdon) [reply index.html]: Possibly. I'll change it and test.

File: index.html (hunk 1 above)
Comment (Windsdon) [reply index.html]: @opl- commented that the spec always uses the full url, so probably a good idea to use it here too. I can move it out of assets, but I think it's nice to have all the images in one place

Comment (opl-) [reply index.html]: Not having it in `/assets` allows for a really fancy URL: `https://minecraftdoublesmoothstoneslab.website/minecraftdoublesmoothstoneslab.jpg`. I'd say lets move it out.

Comment (Windsdon) [reply index.html]: Moved out.

File: index.html (hunk 2 above)
Comment (Windsdon) [reply index.html]: Changed to website.

**Pull request merged**